### ISSUE TITLE
Improve key rendering in documentation

### ISSUE THREAD

**Issue by neiljp** [zulip/zulip-terminal#945]
There are a few improvements we can make in the script which generates `docs/hotkeys.md`; some of the same changes could be added to the rendering of keys in the help menu too.

* [x] space-separated keys (page up/down) are rendered as separate keys in `hotkeys.md` - they should be one key
* [x] we may wish to capitalize or otherwise rename some keys, from those understood by urwid to how they should be rendered
* [ ] in particularly we should consider adding extra symbol text to some keys, eg. ←, ↑, →, ↓  (<kbd>left ←</kbd>?)
* [ ] we might discuss this, but it can be useful to show `P` as <kbd>shift</kbd>+<kbd>p</kbd> (or even <kbd>shift</kbd>+<kbd>P</kbd>?) [noted in #922]

Other ideas are welcome :)

**update** First two points fixed by d65cbee7406a3413946e78fb7fcff8ddc8875251 from #1479,

**Comment by Ezio-Sarthak** [comment]
Apart from the 2nd checkbox (which I don't seem to follow clearly), other improvements seem to be reasonable. I'd happy to push a PR for this :)

**Comment by neiljp** [comment]
The second checkbox is a simpler version of the third really, namely if urwid only generates `tab` but we wished to output `Tab`; the third point might have us adding a tab representation, if that's appropriate, or as indicated there, extra arrow keys.

**Comment by Ezio-Sarthak** [comment]
Makes sense. I seem to have resolved the first part by a one-liner change :)
We might discuss the 4th aspect, as you said, in CZO (others too) in any case :+1:

**Comment by Ezio-Sarthak** [comment]
> The second checkbox is a simpler version of the third really, namely if urwid only generates `tab` but we wished to output `Tab`; the third point might have us adding a tab representation, if that's appropriate, or as indicated there, extra arrow keys.

So, potentially something like `help_keys`, in addition to regular `keys` parameter in `KEY_BINDINGS` ?

**Comment by neiljp** [comment]
Yes, one approach could be to have an extra field, but then we risk having them getting out of sync. I think when discussing this on the stream with @mkp6781 I suggested having a general lookup table for rendering (just hokeys.md, or help too) - that avoids the synchronization issue with another layer of indirection (https://en.wikipedia.org/wiki/Fundamental_theorem_of_software_engineering)

**Comment by neiljp** [comment]
Regarding the fourth point, the webapp takes the `shift + <uppercase visual of key>` approach, though I think there was definitely a time when this was not the case. For hotkeys.md that seems pretty useful, but it seems less obvious within the application since the text is less styled and more obvious? (though of course then we'd have different representations which could be confusing)

**Comment by Ezio-Sarthak** [comment]
@zulipbot add "further discussion required"

**Comment by mkp6781** [comment]
>     * space-separated keys (page up/down) are rendered as separate keys in `hotkeys.md` - they should be one key

Just to clarify. Is this only regarding page up/down or in general. Before having the automation script, most of the space separated keys were rendered separately apart from maybe one or two(like Alt Enter).

**Comment by mkp6781** [comment]
> @mkp6781 Are you planning to take this up? 

You can go ahead with it @Ezio-Sarthak. 👍🏼

**Comment by klfoulk16** [comment]
Regarding the fourth point, I've updated the getting started tutorial to use <shift><p>...shall I update it to include the plus? I feel like previously we didn't use that but now I see all of the hotkeys have that plus.

Are we planning to use this method for all uppercase hotkeys? Originally I know we were discussing P becuase p and P look similar, but s and S (narrowing) also have the same issue.

**Comment by Niloth-p** [comment]
@neiljp I'll make the PR after #1476 is merged.
But, if you've got spare time, please take a look at the commit and the new hotkeys.md in my fork [here](https://github.com/zulip/zulip-terminal/commit/6389a3691e5f9114fd970afbb218643104c9b250) and share your feedback. Eager to get it ready to merge.
